exercises/C-variables/exercise.js

```diff
@@ -1,3 +1,5 @@
 // Start by creating a variable `greeting`
-
+var greeting = "Hello World";
```

Comment:
The same feedback here Shima :) We only need the console.log one time and you can switch `var greeting` to `const greeting` 👍